NLog.Targets.Http/HTTP.cs

```diff
@@ -336,6 +347,10 @@ private void ResetHttpClientIfNeeded()
 
             lock (_propertiesChanged)
             {
+                var oldHttpClient = _httpClient;
```

Comment:
Same here, why create a copy?

Reply:
Because `Dispose` should never throw, but I don't trust other peoples code, so I want the `_httpClient`-variable assigned to zero before calling `Dispose`.